### ISSUE TITLE
init command not working

### ISSUE THREAD

**Issue by vbelovitsky** [qoomon/git-conventional-commits#125]
Hello!
It looks like npm package doesn't contain some necessary files from repository, including [git-conventional-commits.default.yaml](https://github.com/qoomon/git-conventional-commits/blob/master/git-conventional-commits.default.yaml).
So, **init** command not working, because it tries to copy not existing file.
```
Error: ENOENT: no such file or directory, copyfile 'C:\Users\User\AppData\Roaming\npm\node_modules\git-conventional-commits\lib\commands/../../git-conventional-commits.default.yaml' -> './git-conventional-commits.yaml'
```

**Comment by qoomon** [comment]
Damn it. Thanks for reporting. I'll fix it.

**Comment by qoomon** [comment]
should be fixed in version `2.4.1`. thanks again for reporting
